bodhi/server/models.py

```diff
@@ -2571,7 +2576,7 @@ def requested_tag(self):
             # release to the Release.dist-tag
             if self.release.state is ReleaseState.pending:
                 tag = self.release.dist_tag
-        elif self.request is UpdateRequest.testing:
+        elif self.request in (UpdateRequest.testing, UpdateRequest.batched):
```

Comment:
I think this suggestion makes sense. @crungehottman, if you can add one more if statement and one more test for new packages, that would be great!